jobs/generation/Utilities.groovy

```diff
@@ -837,6 +872,38 @@ class Utilities {
 
         addArchival(job, settings)
     }
+    
```

Comment:
You can just eliminate the deprecated version.  We want to move to the ArchivalSettings version when possible.